src/Core/Image/Parser/ImageTagSourceParser.php

```diff
@@ -65,6 +65,6 @@ public function parse($imageTag)
             return null;
         }
 
-        return sprintf('%s%s', $this->shopRootUri, $path[1]);
+        return sprintf('/%s', $path[1]);
```

Comment:
Can we check that this does not break the image display of Languages listing `/admin-dev/index.php/improve/international/languages/` ? 
I dont want to introduce a breaking change for PR https://github.com/PrestaShop/PrestaShop/pull/10864

Reply:
watch out if you're not running under a virtualhost!

Reply:
I have a virtualhost and it doesn't break the image display of Languages listing (if you're talking about the little flag in the language list).

From the behavior I've seen on my local environment:

Both in the vuejs and in the ImageTagSourceParser.php file, the issue was that the shop root uri/ base url was not needed in the image paths. Did I take the problem backward , and the issue is something else?

Here is a correct image path on my local environment:

 http://myprestashop.localhost/img/tmp/product_mini_2.jpg

Reply:
Ok I created a new prestashop environment without a proper virtualhost, and my fix still works, even in the language listing page.

This is the kind of img path I have in this case:

http://localhost/test/PrestaShop/img/tmp/product_mini_1.jpg

Do you have ideas of other things I should watch for?